core/src/main/java/com/linecorp/armeria/internal/common/util/ChannelUtil.java

```diff
@@ -132,10 +143,16 @@ public final class ChannelUtil {
                 ioUringTcpKeepidle = (ChannelOption<Integer>) findChannelOption(clazz, "TCP_KEEPIDLE");
                 //noinspection unchecked
                 ioUringTcpKeepintvl = (ChannelOption<Integer>) findChannelOption(clazz, "TCP_KEEPINTVL");
+
+                tcpOptionsBuilder.add(ioUringTcpUserTimeout);
+                tcpOptionsBuilder.add(ioUringTcpKeepidle);
+                tcpOptionsBuilder.add(ioUringTcpKeepintvl);
```

Comment:
Just out of curiosity; do you know why `KQueueChannelOption` doesn't have these options?